app/controllers/items.js

```diff
@@ -0,0 +1,130 @@
+/**
+  Items Controller
+*/
+
+//3rd party modules
+var express = require('express');
+var mongoose = require('mongoose');
+
+//own modules
+var DefaultController = require('./');
+
+var Item;
+var defaultCtrl;
+
+var Controller = function(){
+  Item = mongoose.model('Item');
+  defaultCtrl = new DefaultController(Item, 'Item');
+  
+  // Define route params
+  this.paramFormat = defaultCtrl.format();
+  this.paramItem = defaultCtrl.modelParam();
+
+  // Define handlers for rest calls
+  this.get = defaultCtrl.get;
+  this.find = defaultCtrl.find;
+  this.create = customCreate;
```

Comment:
these custom functions are more complex than it need to be.. have you think to use e.g mongoose schema validations or e.g json schema here? i think it could simplify these a lot.. 🤔